pkg/protocol/http/http.go

```diff
@@ -358,17 +366,18 @@ func (h *Server) HTTPProcessor(wg *sync.WaitGroup) {
 					ce.SetSubject(d.Status.String())
 
 					if len(h.Publishers) > 0 {
-						if err := Post(fmt.Sprintf("%s/subscription", h.Publishers[0]), *ce); err != nil {
-							log.Errorf("(1)error creating: %v at  %s with data %s=%s", err, h.Publishers[0], ce.String(), ce.Data())
-							log.Errorf("Data sent Address := %s ", d.Address)
-							localmetrics.UpdateSenderCreatedCount(d.Address, localmetrics.ACTIVE, -1)
-							d.Status = channel.FAILED
-							h.DataOut <- d
-						} else {
-							log.Infof("successfully created subscription for %s", d.Address)
-							localmetrics.UpdateSenderCreatedCount(d.Address, localmetrics.ACTIVE, 1)
-							d.Status = channel.SUCCESS
-							h.DataOut <- d
+						for _, pubURL := range h.Publishers {
+							if err := Post(fmt.Sprintf("%s/subscription", pubURL.String()), *ce); err != nil {
```

Comment:
The subscription is broken if not using Publishers[0]/subscription as API URL, since the resource address is passed to pubURL.
```
INFO[0054] posting now SUBSCRIBER, to  /cluster/node/mynode/redfish/v1/Systems/subscription 
INFO[0054] sent: not accepted : false                   
ERRO[0054] (1)error creating: Post "/cluster/node/mynode/redfish/v1/Systems/subscription": unsupported protocol scheme "" at  /cluster/node/mynode/redfish/v1/Systems with data Context Attributes,
```

Reply:
I am using publishers, right now I am looping  through publishers instead of using publishers[0]